docs/site/deployment/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -9,8 +9,8 @@ permalink: /doc/en/lb4/deploying_to_ibm_cloud_kubernetes.html
 
 ## Overview
 
-This tutorial shows how to deploy a [LoopBack4](http://v4.loopback.io/)
-application onto [Kubernetes](http://kubernetes.io/) on the
+This tutorial shows how to deploy a [LoopBack4](http://loopback.io/) application
```

Comment:
```suggestion
This tutorial shows how to deploy a [LoopBack 4](http://loopback.io/) application
```

Reply:
Thanks @achrinza. Perhaps we should change to use `https://loopback.io` as the url everywhere?

Reply:
Sounds like a good idea @dhmlau